lambda/custom/index.js

```diff
@@ -11,8 +11,25 @@ const GAME_LENGTH = 5;
 const SKILL_NAME = 'Reindeer Trivia';
 const FALLBACK_MESSAGE = `The ${SKILL_NAME} skill can\'t help you with that.  It can ask you questions about reindeer if you say start game. What can I help you with?`;
 const FALLBACK_REPROMPT = 'What can I help you with?';
+const APL_DOC = require ('./document/renderPage.json' ) ; 
+const TWO_PAGER_COMMANDS =  require ('./document/twoSpeakItemsCommand.json' ) ;
+const ONE_PAGER_COMMANDS =  require ('./document/oneSpeakItemCommand.json' ) ;
+const TOKEN_ID = 'pagerSample';
+let MAIN_DATA_SOURCE = require ('./document/dataSource.json' );
```

Comment:
I do not think you should modify imported objects.

Reply:
The only one that is modified is MAIN DATA SOURCE (APL Data Source) and that should not be an issue. In the past I included this structure in the index.js file but was asked by Akersh to remove it so the file will be more readable